### ISSUE TITLE
Implement front end interaction for searching across multiple election cycles

### ISSUE THREAD

**Issue by PaulClark2** [18F/fec-cms#1371]
Find out what research Noah did for front-end + design interaction/ interface for filtering multiple election cycles

**Completion criteria**

- [x] find out what Noah did. Old PR work from Noah is here: https://github.com/18F/openFEC-web-app/pull/2213. Need to investigate moving this PR to the newly merged fec-cms application.

Add filter to following data pages:
 - [ ] All receipts: https://www.fec.gov/data/receipts
 - [ ] All disbursements: https://www.fec.gov/data/disbursements
 - [ ] Individual contributions: https://www.fec.gov/data/receipts/individual-contributions

Update Noah's former PR
- [ ]  target the CMS repo, make necessary changes for new environment

**Comment by jenniferthibault** [comment]
@patphongs and @xtine — do you think you could help us get Noah's PR running during the design sync meeting on Fri? Seems like a great use of combined front end/UX time, since I just took a look at the PR description and my eyes bugged out 😳 

Understanding the status of previous work is a blocker to determining if more needs to be done, so would be a big help!

**Comment by patphongs** [comment]
@jenniferthibault Let me give it a shot and see if I can implement it in the new merged app

**Comment by jenniferthibault** [comment]
That would be great. I know you have a lot of other work in progress, so if this puts you over your personal WIP limit, we can recalculate ❤️

**Comment by jenniferthibault** [comment]
Quick update: we took a look at this in design pairing last Friday, and feel good about the initial work already done.

The filter should be added to the following pages, where formerly a two-year restriction was imposed:
- All receipts: https://www.fec.gov/data/receipts
- All disbursements: https://www.fec.gov/data/disbursements
- Individual contributions: https://www.fec.gov/data/receipts/individual-contributions

@patphongs is going to upload a quick screen video of the filter in action for documentation, and carry on moving the PR over from the old repo.

I'll update the completion criteria above and transfer the issue to Pat

**Comment by jenniferthibault** [comment]
When this issue is picked up again, please take a look at https://github.com/18F/openFEC-web-app/issues/2133 , which was rolled into Noah's work on this

**Comment by jenniferthibault** [comment]
**Noting for future github archeology:** 
This issue has not been addressed and and was closed as part of a batch of issues that have been iceboxed for an extended period of time. Implementation of this feature was blocked by poor performance and instability of the feature, seen in https://github.com/18f/openfec/issues/2630
